einops/_backends.py

```diff
@@ -641,4 +641,55 @@ def layers(self):
         return oneflow
 
     def einsum(self, pattern, *x):
-        return self.flow.einsum(pattern, *x)
\ No newline at end of file
+        return self.flow.einsum(pattern, *x)
+
+class MindSporeBackend(AbstractBackend):
+    framework_name = "mindspore"
+
+    def __init__(self):
+        super().__init__()
+        import mindspore as ms
+        ms.set_context(mode=ms.PYNATIVE_MODE)
```

Comment:
Can we use GRAPH_MODE right now ? it seems the very important identity feature in MindSpore.
Thoes who use MindSpore would like to try the jit accelerator.

Reply:
Some pythonic code style or specific grammar is not supported on GRAPH_MODE, like `try...except`, so I have to change the default mode to PYNATIVE. BTW, MindSpore plans to set PYNATIVE mode to support flexible writing, and partial accelerating can use `mindspore.ms_function`.

here is a simple case:
```python
from mindspore import ms_function
from mindspore import nn
class TestNet(nn.Cell):
    def __init__():
        super().__init__():
        self.net = nn.Dense(30, 40)
  
    @ms_function
    def construct(self, x)
        return self.net(x)
```